src/app/stats/stats.component.ts

```diff
@@ -1,4 +1,31 @@
 import { Component, OnInit } from '@angular/core';
+import { BlockTreeService } from './block-tree.service';
+import { ProgressService } from '../progress.service';
+import { BlockTreeResponse } from '../../proto/chain_pb';
+import { Subject, interval } from 'rxjs';
+import { switchMap } from 'rxjs/operators';
+import BigNumber from 'bignumber.js';
+
+interface GraphVertex {
+  id: string | number;
+  label: string;
+  isJustified: boolean;
+  participatedVotes: string;
+  totalVotes: string;
+  numAttestations: number;
+  numDeposits: number;
+  slot: string;
+}
+
+interface GraphEdge {
+  id: string | number;
+  source: string;
+  target: string;
+}
+
+const GENESIS_SLOT = '9223372036854775808';
+const MAX_DEPOSIT_AMOUNT = 3200000000;
```

Comment:
What is special about this number versus `environment.depositAmount`?

Reply:
+1